easybuild/easyconfigs/n/napari/napari-0.4.15-foss-2021a.eb

```diff
@@ -0,0 +1,161 @@
+        'checksums': ['ea0e30fbfdd76542adda44be457f1306a65146fe7578f95838e28d62874f4c45'],
+    }),
+    ('napari-plugin-engine', '0.2.0', {
+        'checksums': ['fa926f869d70e0d652c005661948cd0c7fee5508ae17d437937f34f5287590b3'],
+    }),
+    ('napari-svg', '0.1.6', {
+        'checksums': ['12fa5c64995fc7179349969a39316355496b463858c3b0815e3b3e7ffcdced34'],
+    }),
+    ('Pint', '0.19.2', {
+        'checksums': ['e1d4989ff510b378dad64f91711e7bdabe5ca78d75b06a18569ac454678c4baf'],
+    }),
+    ('build', '0.8.0', {
+        'checksums': ['887a6d471c901b1a6e6574ebaeeebb45e5269a79d095fe9a8f88d6614ed2e5f0'],
+    }),
+    ('psygnal', '0.3.5', {
+        'patches': ['psygnal-0.3.5_python3.8.patch'],
+        'checksums': [
+            '5abf84f0da9f487be68299854c4eb39de0765236407736d201f97ef39913a548',  # psygnal-0.3.5.tar.gz
+            'c55cc07c646cb66e8c5d2ea227249216dbd1a1a9405acbd330d70105a3fd8890',  # psygnal-0.3.5_python3.8.patch
+        ],
+    }),
+    ('pydantic', '1.9.1', {
+        'checksums': ['1ed987c3ff29fff7fd8c3ea3a3ea877ad310aae2ef9889a119e22d3f2db0691a'],
+    }),
+    ('typer', '0.4.1', {
+        'checksums': ['5646aef0d936b2c761a10393f0384ee6b5c7fe0bb3e5cd710b17134ca1d99cff'],
+    }),
+    ('npe2', '0.5.0', {
+        'checksums': ['3d3d43b99bbd3b189fedd7313589c89642a4af0f14b24b9763eb134ba6a5548c'],
+    }),
+    ('numpydoc', '1.4.0', {
+        'checksums': ['9494daf1c7612f59905fa09e65c9b8a90bbacb3804d91f7a94e778831e6fcfa5'],
+    }),
+    ('QtPy', '2.1.0', {
+        'checksums': ['ca8cd4217175186344299ee4c0f7e7adcf362c70852ba35b255a534077025c06'],
+    }),
+    ('superqt', '0.3.2', {
+        'checksums': ['a37df244a2d6e4387e865cdedabe925798b3352ab9cdc52f60701eb12d907960'],
+    }),
+    ('tifffile', '2022.5.4', {
+        'checksums': ['b03147a15862b7c1d90d47435197f149bef7a52c25ad67cf1f9b465faa71b8d2'],
+    }),
+    ('pandas', '1.3.0', {
+        'checksums': ['c554e6c9cf2d5ea1aba5979cc837b3649539ced0e18ece186f055450c86622e2'],
+    }),
+    ('beniget', '0.4.1', {
+        'checksums': ['75554b3b8ad0553ce2f607627dad3d95c60c441189875b98e097528f8e23ac0c'],
+    }),
+    ('ply', '3.11', {
+        'checksums': ['00c7c1aaa88358b9c765b6d3000c6eec0ba42abca5351b095321aef446081da3'],
+    }),
+    ('gast', '0.5.3', {
+        'checksums': ['cfbea25820e653af9c7d1807f659ce0a0a9c64f2439421a7bba4f0983f532dea'],
+    }),
+    ('pythran', '0.10.0', {
+        'checksums': ['9dac8e1d50f33d4676003e350b1f0c878ce113e6f907920e92dc103352cac5bf'],
+    }),
+    ('PyWavelets', '1.2.0', {
+        'modulename': 'pywt',
+        'checksums': ['6cbd69b047bb4e00873097472133425f5f08a4e6bc8b3f0ae709274d4d5e9a8d'],
+    }),
+    ('imread', '0.7.4', {
+        'checksums': ['0487adef11a22168700968c1727020361a72f6132b6ced2b8826b02d8cbf744f'],
+    }),
+    ('pooch', '1.5.2', {
+        'checksums': ['5969b2f1defbdc405df932767e05e0b536e2771c27f1f95d7f260bc99bf13581'],
+    }),
+    ('scikit-image', '0.19.1', {
```

Comment:
add a dependency on one of the `scikit-image` easyconfigs instead, and remove packages from the extensions here that are provided by that easyconfig

Reply:
napari requires scikit-image 0.19.1. For foss/2021a we have 0.18.3
it's probably best to switch to a newer toolchain.